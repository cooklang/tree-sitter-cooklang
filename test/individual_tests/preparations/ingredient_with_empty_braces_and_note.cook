Season with @pepper{}(freshly ground).