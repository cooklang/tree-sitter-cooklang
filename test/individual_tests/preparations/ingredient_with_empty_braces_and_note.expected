(recipe 
  (step 
    (ingredient 
      (name)
      note: (note_text)))